### DISCUSSION TITLE
How do you download the files here?

### DISCUSSION THREAD

**Discussion by vazquez595654** [bluefireteam/audioplayers#1616] [Q&A]
Hi,
There is no download link for the example project. How do you download all the project files, or at least the folders?

I am downloading each dart file one by one, and it is very painful. For an example project, this project uses so many files. Shouldn't it be able to be made with just one dart file, for an example?

**Comment by Gustl22** [comment]
To show all the functionality of audioplayers, we have a project which has quite a lot of code. To avoid a mess, it consists of multiple files.

Consider downloading the whole git repository and try to run the example project in the according subdirectory.

Note if you use the main version, you have to update the dependencies inside the monorepo:
https://github.com/bluefireteam/audioplayers/blob/main/contributing.md#environment-setup

The more basic instructions, you can see here:
https://github.com/bluefireteam/audioplayers/blob/main/getting_started.md

But I agree, we may add an example.md with a more basic setup.
